source/common/http/http3/conn_pool.cc

```diff
@@ -65,10 +65,13 @@ allocateConnPool(Event::Dispatcher& dispatcher, Random::RandomGenerator& random_
       host, priority, dispatcher, options, transport_socket_options, random_generator, state,
       [&quic_stat_names,
        &scope](HttpConnPoolImplBase* pool) -> ::Envoy::ConnectionPool::ActiveClientPtr {
+        pool->addLog(Logger::Logger::debug, "Create Http/3 client");
```

Comment:
Why not log these directly/  Does the Http3 Pool just need to inherit from
 : protected Logger::Loggable<Logger::Id::pool>

Reply:
This is in the lambda rather than in the pool instance.

Reply:
Ah check.  I think the canonical way to handle that is ENVOY_LOG_TO_LOGGER() rather than passing the logger in.
Mind giving that a shot, and if you think it's uglier we can go with this as-is?

Reply:
Yeah, I already switch to that in a latter commit.

Reply:
yeah but you're still passing spdlogs around
can't you
ENVOY_LOG_TO_LOGGER(Envoy::Logger::Registry::getLog(Envoy::Logger::Id::pool) 
instead of changing the signatures to pass the logger around?

Reply:
done

Reply:
I'm sure this is the right approach, but I don't think I understand envoy logging well enough. What does `Envoy::Logger::Registry::getLog(Envoy::Logger::Id::pool)` do?

Reply:
Get the spd::logger object associated with pool. It should have been registered at construction of Loggable<pool> object.